config/initializers/slowweb.rb

```diff
@@ -1,4 +1,4 @@
 require 'slowweb'
 # todo: there is a bug where this is only called half as many times
-limit = ENV.fetch('OSCN_REQUESTS_PER_MINUTE', 6)*2
+limit = ENV.fetch('OSCN_REQUESTS_PER_MINUTE', 12).to_i*2
```

Comment:
it seems that here you change  the request per minute to be 12 instead of 6 and made sure to convert this environment variable to an integer before multiplying it by 2. interesting!